package.json

```diff
@@ -0,0 +1,70 @@
+{
+    "name": "ox",
+    "description": "Please enter a brief description here",
+    "version": "0.0.1",
+    "files": [
+        "bin"
+    ],
+    "type": "module",
+    "scripts": {
+        "build": "tsc -b tsconfig.json && node esbuild.mjs",
+        "watch": "concurrently -n tsc,esbuild -c blue,yellow \"tsc -b tsconfig.json --watch\" \"node esbuild.mjs --watch\"",
+        "lint": "eslint src --ext ts",
+        "langium:generate": "langium generate",
+        "langium:watch": "langium generate --watch",
+        "vscode:prepublish": "npm run build && npm run lint"
+    },
+    "dependencies": {
+        "langium": "~2.0.1",
+        "llvm-bindings": "~0.4.2",
```

Comment:
@pluralia  `npm i` does not work for me locally, same in GitPod ... probably, since the dependency `llvm-bindings` is missing. Or do I missed something?

Reply:
@JohannesMeierSE, thanks, I removed this dependency

Reply:
Thanks, it works for me!